src/components/MoreInfoModal/MoreInfoModal.tsx

```diff
@@ -0,0 +1,147 @@
+/* eslint-disable react/jsx-props-no-spreading */
+/* eslint-disable react/no-array-index-key */
+import 'react-slideshow-image/dist/styles.css';
+
+import CloseIcon from '@mui/icons-material/Close';
+import FavoriteIcon from '@mui/icons-material/Favorite';
+import { Paper } from '@mui/material';
+import Button from '@mui/material/Button';
+import Modal from '@mui/material/Modal';
+import Typography from '@mui/material/Typography';
+import * as React from 'react';
+import { Slide } from 'react-slideshow-image';
+import photosHouse from 'src/assets/images/House.png';
+import photosMap from 'src/assets/images/Map.png';
+import photosNavLogo from 'src/assets/images/NavLogo.png';
+import photosNoPhoto from 'src/assets/images/nophoto.png';
+
+import FacilitiesList from './FacilitiesList';
+import styles from './MoreInfoModal.module.scss';
+import PropertyInformationList from './PropertyInformationList';
+
+const style = {
+  p: 2,
+  pb: 4,
+};
+
+const houseImages = [photosHouse, photosNavLogo, photosMap, photosNoPhoto];
+
+type HouseInfo = {
+  url: string;
+  caption: string;
+}[];
+
+// Create array with house pohotos for modal
+const slideImages: HouseInfo = [];
+
+houseImages.forEach((houseImage, index) => {
+  slideImages.push({
+    url: houseImage,
+    caption: `Photo ${index}`,
+  });
+});
+
+const slideProperties = {
+  autoplay: false,
+};
+
+const moreInfo = {
+  address: '420 Baker St, London',
+  price: '$299,999',
+};
+
+const propertyInformation = {
+  propertyType: 'House',
+  area: '1200',
+  yearBuilt: '2000',
+  floor: '1',
+  floorsInBuilding: 'House',
+  roomsNumber: '5',
+  bathroomNumber: '2',
+  heating: 'Radiators',
+};
+
+export default function BasicModal() {
+  const [open, setOpen] = React.useState(false);
```

Comment:
`const [open, setOpen] = React.useState(false);` won't probably stay here when the modal is placed in the JSX structure of the home page from where it will be opened so as part of this taks please enable opening the modal by clicking the buttons
![Screenshot from 2022-04-18 22-16-08](https://user-images.githubusercontent.com/47178494/163871123-17752fc7-8ca6-455d-a9fe-4423f032c25d.png)
on the house list on the home page.

Reply:
This "open" state is used to decide whether the modal is visible or not, so I think it will stay in this file.

I am pushing a version of the modal that opens when You click on the "More info" button in the ListOfHouses component.
It leaves the `const [open, setOpen] = React.useState(false);` and everything works nicely.
Please - let me know what You think about it.

After that, the only thing left is passing props from the given house to the modal.

Reply:
Just pushed it in the last commit ;)

Reply:
Works great! Well done :rocket:!

No need to pass props to the modal as for the modal to have all the necessary data, a request to the server is needed because data about houses available on the houses list is by design basic enough to make only the list and the map work.